src/ucm/bistro/bistro_loongarch64.c

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (C) Tactical Computing Labs, LLC. 2022. ALL RIGHTS RESERVED.
```

Comment:
Please fix copyright for all files

Reply:
Seems copyrights should be named after Loongson company not Tactical Computing Labs?